tests/integration/high_availability/high_availability_helpers.py

```diff
@@ -106,7 +106,8 @@ async def ensure_n_online_mysql_members(
                 assert len(online_members) == number_online_members
                 return True
     except RetryError:
-        return False
+        pass
+    return False
```

Comment:
why this change? it is equivalent of `return False` from the except block

Reply:
just to make the linter happy